proxy/list_test.go

```diff
@@ -30,7 +30,7 @@ func Test_ListFunctions(t *testing.T) {
 	}
 	for k, v := range result {
 		if expectedListFunctionsResponse[k] != v {
-			t.Fatal("Expeceted: %#v - Actual: %#v", expectedListFunctionsResponse[k], v)
+			t.Fatalf("Expeceted: %#v - Actual: %#v", expectedListFunctionsResponse[k], v)
```

Comment:
We have a typo here. Also the Go test output should be want and got rather than expected actual. Can you update please?